feat(content): The Successors - Enter the Frogtopus Aristocracy (#10296)
endless-sky/endless-sky: data/successors/successor side missions.txt

# Copyright (c) 2024 by Daeridanii
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
```

				decline
			label hidden
			`	You hold the stick behind your back until you head back inside the <ship> and place it in a waste receptacle.`
				decline

mission "Successors: First Contact Exploration: Iyra-Ijasa-Iret"
	minor
	source "Iyra-Ijasa-Iret"
	to offer
		has "Successors: First Contact 1: done"
		not "Successors: First Contact 2: done"
	on offer
		conversation
			`You've never seen a spaceport that's entirely underwater before. Next to your landing pad, there's a massive window into what you might call an aquarium but what seems to be some sort of lounge for the Successors, who swim around and organize themselves around several multi-layered tables.`
				decline

mission "Successors: First Contact Exploration: Mavra-Sol-Kvel"
	minor
	source "Mavra-Sol-Kvel"
	to offer
		has "Successors: First Contact 1: done"
		not "Successors: First Contact 2: done"
	on offer
		conversation
			`Your ship draws a bit of a crowd when you land here, though most are kept at bay by a set of pre-arranged barriers set up around your landing pad and the occasional patrolling guard. While walking by the barrier, one Successor offers one of their data-spheres to you.`
			choice

> 				`	(Take it.)`

Reviewer 2: ```suggestion
				`	(Take it.)`
					goto taken
```

				`	(Refuse.)`

> 					goto refused

Reviewer 2: ```suggestion
					goto refused
			label taken
			action
				outfit "Local Map" 1
```

			`	The Successor trills warmly when you take the sphere, though they are quickly pushed back by one of the guards. Like previously, the sphere expands into a small star map as soon as you touch it, though this one seems to contain some cultural data in addition to stellar coordinates.`
				decline

> 			action

Reviewer 2: ```suggestion
```

> 				outfit "Local Map" 1

Reviewer 2: ```suggestion
```

			choice
				`	"Sure, I would be happy to."`
					goto accept
				`	"Can I come back another time?"`
					defer
				`	"Sorry, but you'll have to find someone else to take you."`
					decline
			label accept
			`	"Our deepest thanks!" Several of the members of the research team smile conspiratorially, clearly excited to be skipping the queue. Sieasej also offers their thanks and makes clear that they'll "keep an eye out for any more interesting opportunities."`
				accept
	on stopover
		conversation
			`While relatively calm and subdued during the journey here, the researchers bolt out of the airlock almost the instant your ship's landing gear touch ground, arms loaded down with an innumerable array of booklets, catalogs, and strange devices that they waste no time in thrusting into the air or towards the rocks and soil.`

> 			`	By the time night falls on the planet, they reluctantly return to your ship, dusty, tired, but abuzz with excitement at their discoveries.`

Author: I wasn't terribly fond of "when" here, but I rephrased this section---see if you like it better now.

> 			`	By the time night falls on the planet, they reluctantly return to your ship, dusty, tired, but abuzz with excitement at their discoveries.`

Reviewer 2: Looks good.

					goto empire
				`	"What was their technology like?"`
					goto tech
				`	"I have no more questions."`
					launch
			label collapse
			`	The researcher sighs. "We do not know; or better put, the fullness of the truth yet remains obscured. The flesh of space itself was rent, distorted; the effects of this remain today: the holes embedded in the skies, the dying-out of light and even of the hyperlanes. It was a warring time; the Houses turned upon each other or they fled, and space itself was set alight with the blinks of shots and drives---"`
			`	The researcher stops for a moment, composing themself.`
			`	"I digress and do apologize. T'is challenging to separate the truth from fiction in this regard. Our Predecessors wielded power far beyond our modern imitations, the sort that sows the seeds of peace or chaos deep within the fabric of the world, the sort of things that poke and prod at places that must remain untouched. 'What caused the collapse?' you ask? Hubris, methinks."`
			choice
				`	"How large was their empire?"`
					goto empire
				`	"What was their technology like?"`
					goto tech
				`	"I have no more questions."`
					launch
			label empire

> 			`	"The translation device thou wears upon thy person speaks volumes to its scale. Sieasej tells me that at that empire's peak, our scholars had even laid eyes upon thy own world, far to the north! These dead systems that remain where once the core of their empire stood are but a small fraction of what was there. Some, it's sure, now belong to other races, and others, it would seem, have been obscured. We know not how nor why, only that the starcharts of our Predecessors do not mirror those we see today."`

Reviewer 2: ```suggestion
			`	"The translation device thou wears upon thy person speaks volumes to its scale. Sieasej tells me that at that the empire's peak, our scholars had even laid eyes upon thy own world, so far to the north! These dead systems that remain where once the core of their empire stood are but a small fraction of what was there. Some, it's sure, now belong to other races, and others, it would seem, have been obscured. We know not how nor why, only that the starcharts of our Predecessors do not mirror those we see today."`
```

			choice
				`	"What caused the collapse?"`
					goto collapse
				`	"What was their technology like?"`
					goto tech
				`	"I have no more questions."`
					launch
			label tech

> 			`	"Glorious and beautiful; as much art as science or engineering. At their peak, the works of our Predecessors were envied and respected even by the Quarg, whose mastery knows little equal. Twere exotic in their natures and unique in applications; living metal grown in stars, subtle points to pierce space itself. We have descriptions, enough to make rudimentary imitations of their processes, but even if we had a perfect understanding of wha'twas, the space of our new systems seems unsuitable to manifest that mastery. We fail even to replicate the heirlooms kept by the High Houses; the materials and techniques they use are always out of reach."`

Reviewer 2: That first part does feel somewhat awkward, though I'm not sure my suggested change makes it much better.
```suggestion
			`	"Glorious and beautiful; as much art as science or engineering. At their peak, the works of our Predecessors were envied and respected even by the Quarg, whose mastery only few equal. T'were exotic in their natures and unique in applications; living metal grown in stars, subtle points to pierce space itself. We have descriptions, enough to make rudimentary imitations of their processes, but even if we had a perfect understanding of wha'twas, the space of our new systems seems unsuitable to manifest that mastery. We fail even to replicate the heirlooms kept by the High Houses; the materials and techniques they use are always out of reach."`
```

mission "Successors: Biology Research 1"
	minor
	name "Assist Predecessor Biologist"
	description "Transport a Successor biologist to <stopovers> and then home to <destination> by <date>."
	passengers 1
	deadline 20
	to offer
		has "known to the successors"
		random < 30
	source
		government "Successor" "Old Houses" "New Houses"
	stopover "Naph-Naap-Kella"
	destination "Staja-Kella-Oa"
	on offer
		conversation
			`While you are sitting in the spaceport, you are approached by a Successor wearing a curious green outfit and carrying several pieces of seemingly high-tech equipment, enough that it appears close to falling out of their arms. They explain that they are a Successor biologist who had chartered a flight to go and examine the primordial life on Naph-Naap-Kella, a young and still-developing planet, but that their pilot had been detained, leaving them stranded here.`
			`	"If thou coulds't provide travel to that planet, I may still be able to gather my data before my period of research leave expires."`
			choice
				`	"Sure, I would be happy to."`
					goto accept
				`	"Sorry, but you'll have to find someone else to take you."`
					decline
			label accept

> 			`	"Thou has my deep and certain thanks. Tell none, but the High Houses are beastly to us academics." The biologist, who introduces themselves as "Kaatiora", loads their collection of equipment into your ship---though not without almost dropping one particularly fragile-looking item---and is eventually ready to leave.`

Reviewer 2: Maybe?
```suggestion
			`	"Thou has my deep and certain thanks. Tell none, but the High Houses are beastly to us academics." The biologist, who introduces themselves as "Kaatiora," loads their collection of equipment into your ship---though not without almost dropping one particularly fragile-looking item---and is eventually ready to leave.`
```

				accept
	on stopover
		conversation

> 			`On the way here, Kaatiora has been impressively inquisitive, continually asking you questions about human biology with a level of verve and excitement you though previously impossible. Once they exhausted your knowledge of every detail of human biology, they moved on without hesitation to every other Earth-related species they knew of from Predecessor records, from cattle to frogs to far more exotic ones that you've never heard of in your life but apparently are all the rage among the circles of Successor xenobiology.`

Reviewer 2: ```suggestion
			`On the way here, Kaatiora has been impressively inquisitive, continually asking you questions about human biology with a level of verve and excitement you had previously thought impossible. Once they exhausted your knowledge of every detail of human biology, they moved on without hesitation to every other Earth-related species they knew of from Predecessor records, from cattle to frogs to far more exotic ones that you've never heard of in your life but apparently are all the rage among the circles of Successor xenobiology.`
```

			`	It is somewhat relieving when you finally arrive at the planet and Kaatiora momentarily departs, once again carrying their more-than-armful of equipment and excitedly but carefully scanning the volcanic hot-water pools and small lakes around your landing site, occasionally exclaiming with excitement.`
			`	Eventually they return to your ship, satisfied with the amount of data they have collected, but still obviously energized by the experience.`
			`	Before leaving, you realize this would be a good time to learn more about the Successors from an expert. Besides, since you spent the majority of the time on the way here answering Kaatiora's questions, you are probably entitled to ask a few of your own.`
			choice

> 				`	(Ask about the Successors and biology.)`

Reviewer 2: This doesn't feel great, though I'm not sure what else to put there.

Reviewer 2: Looks good now.

					goto ask

> 				`	(Don't ask them.)`

Reviewer 2: ```suggestion
				`	(Don't ask anything.)`
```

					launch
			label ask
			`	As expected, Kaatiora is more than happy to share their biological knowledge, and asks if there is anything in particular you'd like to know about.`
			choice
				`	"What can you tell me about the Successors?"`
					goto successor
				`	"Were the Predecessors any different, biologically speaking, from the present Successors?"`
					goto predecessor
				`	"What is the life in this star cluster like?"`
					goto cluster
				`	"I have no more questions."`
					launch
			label successor
			`	"Much of our evolutionary history, we spent beneath the waves of our home planet, and while now you clearly see we breathe the same air as you, we still retain our cold and water-loving tendencies. Our respiration works just as well submerged as on dry land, though we must retain some degree of moisture on our skin.`

> 			`	Apart from that, we are a hermaphroditic species, like most from our home planet, and as we took to the stars, we learned that this is uncommon but not rare, and so there is some curiosity as to why our planet had such a diversity of such species while most do not. Perhaps, since we grew up under a harsh sun, the redundancy provided by our divorce from biological sex was beneficial to our evolution.`

Reviewer 2: The first line may have too many commas?
```suggestion
			`	"Apart from that, we are a hermaphroditic species, like most from our home planet, and as we took to the stars, we learned that this is uncommon but not rare, and so there is some curiosity as to why our planet had such a diversity of such species while most do not. Perhaps, since we grew up under a harsh sun, the redundancy provided by our divorce from biological sex was beneficial to our evolution.`
```

> 			`	And of course, thou hast surely noticed by now the natural camouflage of our skin. While today its use its for communication, it was long ago a useful tool for evading predators who hunted by sight, until, of course, we gained the wisdom to outsmart our predators rather than hide from them."`

Reviewer 2: ```suggestion
			`	"And of course, thou hast surely noticed by now the natural camouflage of our skin. While today its use is for communication, it was long ago a useful tool for evading predators who hunted by sight, until, of course, we gained the wisdom to outsmart our predators rather than hide from them."`
```

			choice

> 				`	"Were the Predecessors any different, biologically speaking?"`

Reviewer 2: ```suggestion
				`	"Were the Predecessors any different, biologically speaking, from you?"`
```

					goto predecessor
				`	"What about the other life in this star cluster?"`
					goto cluster
				`	"I have no more questions."`
					launch
			label predecessor
			`	"As brevity is the soul of wit, then no, our Predecessors did not differ greatly from us in their core biology. A scion of the High Houses from the time of the collapse would not stand out from the average one of us in the present day. That being said, their understanding of it was much superior: their medical science was much more advanced than our own, and much of their technology was genetically coded to function only for specific families."`
			choice

> 				`	"What can you tell me about modern Successors?"`

Reviewer 2: ```suggestion
				`	"What can you tell me about the modern Successors?"`
```

					goto predecessor
				`	"I have no more questions."`
					launch
	on complete
		"successor jobs" += 2
		payment 200000
		conversation
			`	Kaatiora remains excited and talkative on the way back, though marginally less so now that they have exhausted the totality of your knowledge of the biological sciences and have the analysis of their recently-gathered data to occupy them.`
			`	Upon landing at <destination>, they disembark from your ship, once again carrying their excess of equipment, but not before giving you your payment of <payment> and once again offering their thanks for transporting them.`

mission "Successors: Buskers"
	minor
	source
		government "Successor"
		not attributes "crime" "quiet"
	to offer
		has "known to the successors"
		random < 40
	on offer
		conversation
			`While waiting in the spaceport for the <ship> to be refueled, you notice that a small crowd of Successors is gathering around one of the pools of water that surround the starship hangars. Many are moving excitedly towards the commotion or, having already made it there, undulate strangely in the water.`
			choice
				`	(Investigate the gathering.)`
				`	(Ignore it.)`
					decline

> 			`	You walk through the water-covered floor to the group, and as you start to grow closer, you begin to hear music coming from the pool: a strange, lilting melody filled with notes that seem impossibly high or low or both at once. As you squeeze your way through the crowd, you see that in the pool there is a trio of submerged Successors, each of whom has their arms wrapped around one or several wildly distinct metal devices which protrude through the surface of the water. It is from these devices that the music emanates.`

Reviewer 2: ```suggestion
			`	You walk over the water-covered floor to the group, and as you start to grow closer, you begin to hear music coming from the pool: a strange, lilting melody, which is filled with notes that seem impossibly high or low or both at once. As you squeeze your way through the crowd, you see that in the pool there is a trio of submerged Successors, each of whom has their arms wrapped around one or several wildly distinct metal devices which protrude through the surface of the water. It is from these devices that the music emanates.`
```

			`	One of the Successors in the pool begins to sing; at least, singing is the best way you could describe it: they press one of the longer and more cylindrical instruments up to their head and it begins producing an accompaniment to the other instruments' melody that your translation device occasionally recognizes as words:`

> 			`	"... sweetest child ... never want ... not so ... somber current ... endure ..."`

Reviewer 2: Sorry if I've suggested this before.
```suggestion
			`	"... sweetest child... never want... not so... somber current... endure..."`
```

					decline

			label applaud
			`	The Successors around you seem a little surprised or perhaps confused by your display, but do not express any offense at it, as far as you can tell. The trio of performers eventually emerge from the pool, gently tucking their instruments into various equally oddly-shaped cases before chatting with some of the few remaining members of the crowd and then departing.`
				decline

			label hum
			`	The Successors around you seem amused by your attempt to emulate the humming; some briefly display a bluish-green wave of color across their skin before quickly returning to their base shade. Nonetheless, no one here seems to take any significant offense at the gesture. The performers eventually emerge from the pool, gently tucking their instruments into equally oddly-shaped cases before chatting with some of the few remaining members of the crowd and then departing.`
				decline

			label unappreciative
			`	You return to your ship, gently extricating yourself from the throng of spellbound Successors, who continue to shimmer and sway in the distance until the performance eventually ends and they disperse to go their separate ways.`
				decline

> mission `Successors: Stargazing 1`

Reviewer 2: ```suggestion
mission "Successors: Stargazing 1"
```

	minor
	name "Transport Successor stargazers"

> 	description "Transport a pair of Successor stargazers to <stopovers> and return them home to <destination> by <date>."

Reviewer 2: What do you think? [Here](https://github.com/Daeridanii1/successors/pull/14#pullrequestreview-2142340636) you were saying to do `<planet stopovers>`, but that maybe shouldn't be the case here?
```suggestion
	description "Transport a pair of Successor stargazers to <planet stopovers> and return them home to <destination> by <date>."
```

			`	"And back, of course. And our cargo," the other one says, gesturing towards the cart of folded fabric.`
			`	"And back. And our cargo. We can give thee modest pay."`
			choice
				`	"Sure, I can take you."`
					goto yes
				`	"You'll have to find a different pilot."`
					decline
			label immediate
			`	"Oh! 'Tis perfect! We were looking for a private transport. We're stargazers---"`
			`	"There's a planet at the edge of the galaxy with the perfect view---"`
			`	"The sky there's divided into star-studded night and utter void. Could thou take us there?"`
			`	"And back, of course. And our cargo too," the other one says, gesturing towards the cart of folded fabric.`
			`	"And back. And our cargo. We can give thee modest pay."`
			choice
				`	"Sure, I can take you."`
					goto yes
				`	"I'm sorry, but you'll have to find a different pilot."`
					decline
			label yes
			`	The two seem thrilled, the larger one turning a dull pink. "Oh! Thour't truly honorable, noble alien. I am Nnesa."`
			`	"Jaasora. Thank you, Captain. Most pilots here've no appetite to travel to the uninhabited systems."`
			`	"Yes, you must tell us about all the places you've been---"`
			choice

> 				`	"How about we get you boarded first."`

Reviewer 2: ```suggestion
				`	"How about we get you onboard first."`
```

			`	Khosa-Kella-Oa itself is cold, dark and rocky, for all purposes functionally identical to any of the other innumerable uninhabited airless worlds in the galaxy. Your passengers seem pleased with it, though; Nnesa remarks that the local star "is so dim that there's hardly any glare!" and Jaasora energetically agrees. They activate their cart of folded equipment within the <ship>'s cargo bay, which slowly inflates and unfolds into something halfway between a tent and a repulsor lift. It skids over the floor like melting ice, gently bumping the inside walls a couple times before Jaasora manages to quiet its movement.`
			`	"Our thanks again, captain, for transporting us," says Jaasora. They load a bundle of telescopes into the tent.`
			`	"Oh! The planet's perfect; no air to muddy the view nor other ships to cloud the skies---"`
			`	"We should be back in a couple hours. Please wait for us."`
			`	The pair depart in their gliding tent, which manifests several transparent spots on its surface as it exits and as a bevy of telescopes are prodded around inside.`
			choice

> 				`	(Wait.)`

Reviewer 2: Should there be an option here to leave that makes the Successors angry at you for good, or did you decide that it probably isn't good for people to be locked out this early?

Author: I suppose it struck me as so cruel an option it never crossed my mind to add it as a choice. I'll check if there's precedent for the player being given these sorts of choices, and if so, I suppose I can add it.

Reviewer 3: I feel that a "leave without them" option's most likely use case is "fuck I accidentally clicked it why is that even there"....

> 			`	Nnesa and Jaasora return after a couple of hours, as promised, their tent gliding back into the <ship>'s cargo bay as smoothly as it exited. The pair unload several notebooks of photos and drawings and chatter excitedly about the various constellations and stellar phenomena that they observed. Nnesa is particularly excited, hurriedly extracting from the interleaved mess of papers a surprisingly accurate and well-constructed star map, which they press up in your face with pride. You cannot help but notice that several of the drawings in the stack are of Jaasora rather than of any stellar phenomena.`

Reviewer 2: Tent? What for?
Maybe you should specify that it is the thing that looks like a tent?

			`	"From where on the map dost thou hail, captain?" Nnesa roughly gestures to an area of their map that you recognize as human space.`
			choice

> 				`	(Indicate Rutilicus.)`

Reviewer 2: These options will need to be added to in the future, especially when #7721 is merged.

					goto rutilicus
				`	(Indicate Sol.)`
					goto sol
				`	(Gesture broadly to the expanse of space depicted on the map.)`
			`	"Oh!" says Nnesa, "Thou art a scion of the constellations, first and foremost to the stars? 'Tis fair. Thou hast a galaxy to wander, a nearly endless sky. The things that there must be to see---"`
				goto interrupt
			label rutilicus
			`	"Oh!" says Nnesa, "I am not sure I know the name for that one, captain. What dost thou call thy home?"`
			choice
				`	"Rutilicus."`
					goto rutilicus2
			label rutilicus2
			`	Nnesa asks you to repeat the word a second time, slowly working their way through your pronunciation and transcribing a short sequence of flowing letters on their map. "'Tis a pleasant name," they say.`
				goto interrupt
			label sol

> 			`	"Oh!" says Nnesa, "Sol." They quietly sound their way through the word a couple times, trying to get it right. "Our name for it is Saaqa-Viirja. 'Twas one of the most-northern extents of our Predecessors' voyages. Ah. But thou know'st this already, for sure."`

Reviewer 2: ```suggestion
			`	"Oh!" says Nnesa, "Sol." They quietly sound their way through the word a couple times, trying to get it right. "Our name for it is Saaqa-Viirja. 'Twas one of the northernmost extents of our Predecessors' voyages. Ah. But thou know'st this already, for sure."`
```

					goto ignored
			`	Upon approaching the source of the clattering, you catch a trio of Successors in the process of smearing the hull of the <ship> with globs of a colorful, foul-smelling solution. It is so pungent you can barely approach them without retching, but the Successors seem not to notice, or not to care.`
			choice
				`	"What are you doing to my ship?"`
					goto what
				`	"Excuse me, are you all part of the repair crew?"`
			`	The three Successors startle at the sound of your voice, arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "Yes, correct thou is! We've orders to apply this ... hull solution to repair a spot of damage here."`
			`	Waves of color run up the bodies of the others, and the tips of their arms undulate gently.`
			choice
				`	"I've never seen a solution like that before. How does it work?"`
					goto how
				`	"I'm sorry to interrupt in that case. Carry on."`
					goto fooled

			label what

> 			`	The three Successors startle at the sound of your voice, their arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "We ... ah ... we're part of the repair crew! We've orders to apply this ... hull solution to repair a spot of damage here."`

Reviewer 2: ```suggestion
			`	The three Successors startle at the sound of your voice, their arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "We... ah... we're part of the repair crew! We've orders to apply this... hull solution to repair a spot of damage here."`
```

			`	Waves of color run up the bodies of the others, and the tips of their arms undulate gently.`
			choice
				`	"I've never seen a solution like that before. How does it work?"`
					goto how
				`	"I'm sorry to interrupt in that case. Carry on."`
					goto fooled

			label how

> 			`	The group look around at each other before the same one replies again. "The ... ah ... solution seeps into the cracks and strains within the hull and when it dries it seals them. Beyond that, we do not know; we just apply it as directed by the port authority."`

Reviewer 2: ```suggestion
			`	The group look around at each other before the same one replies again. "The... ah... solution seeps into the cracks and strains within the hull and when it dries it seals them. Beyond that, we do not know; we just apply it as directed by the port authority."`
```

			`	One of the other ones pipes up, cheerfully adding, "We don't know anything!" before the others give them a dark look.`
			choice
				`	"And it's so colorful too! Are you sure it's not a paint of some sort?"`
					goto colorful
				`	"Fascinating. Carry on."`
					goto fooled

			label colorful

> 			`	"Oh, that's for ... ah ... safety. Yes, 'tis very dangerous to touch if thou'r'nt trained, so 'tis bright in color as a warning." None of the three are wearing any sort of protective equipment that you can see, and by now most of the group have changed color to what you might characterize as a mirthful shade of pink. This is something which one or two are clearly trying to suppress, their skin shifting in gradients in an effort to maintain their previous dull purple hue. One of the others is trying to hide a bag filled with more cans of "hull solution" behind their arms.`

Reviewer 2: ```suggestion
			`	"Oh, that's for... ah... safety. Yes, 'tis very dangerous to touch if thou'r'nt trained, so 'tis bright in color as a warning." None of the three are wearing any sort of protective equipment that you can see, and by now most of the group have changed color to what you might characterize as a mirthful shade of pink. This is something which one or two are clearly trying to suppress, their skin shifting in gradients in an effort to maintain their previous dull purple hue. One of the others is trying to hide a bag filled with more cans of "hull solution" behind their arms.`
```

				`	"Nice try, but you and I both know what's going on here."`
					goto called
				`	"I think my ship's hull has been well enough repaired. It might be best for you to attend to whatever other vessels are on your list."`
					goto complicit
				`	"Thanks for answering my questions. Please carry on."`
					goto fooled

			label called
			`	The trio seem rather deflated at hearing this, but after quite a bit of milling amongst themselves eventually relent and agree to remove the solution so long as you don't inform the port authority about their actions. While they put good effort into removing it---or at least act the part---they can't seem to scrub away the most dried-on parts, and you eventually let them go after it becomes apparent that some of the paint has permanently bonded to the hull.`
				decline

			label complicit
			`	The trio brighten up at hearing this, and after some milling amongst themselves eventually glide off into the spaceport with their cans of solution in tow, though not before leaving one with you for "emergency repairs," though considering how terrible it smells it's not certain if you could ever stand trying to apply it.`
				decline

			label ignored

> 			`	Upon returning from the spaceport some time later, you find that an area on the side of the <ship> has been covered in a thick, colorful solution with an incredibly noxious smell so potent you can hardly bear to approach it. The pattern forms some words in the Successor script, and while the translation device struggles to interpret the non-standard glyphs, it eventually offers a translation along the lines of "Ryiaa was here (and friends)." The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`

Reviewer 2: Looks like I missed something in #[19](https://github.com/Daeridanii1/successors/pull/19).
```suggestion
			`	Upon returning from the spaceport some time later, you find that an area on the side of the <ship> has been covered in a thick, colorful solution with an incredibly noxious smell so potent you can hardly bear to approach it. The pattern forms some words in the Successor script, and while the translation device struggles to interpret the non-standard glyphs, it eventually offers a translation along the lines of "Ryiaa was here (and friends.)" The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`
```

				decline

			label fooled
			`	Upon returning from the spaceport some time later, you find that the "repair crew" has covered an area on the side of the <ship> in their colorful and noxious solution. The pattern forms some words in the Successor script, and while the translation device struggles to interpret the non-standard glyphs, it eventually offers a translation along the lines of "Hai-Fool! Naked with no tail." The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`

> 				decline

Reviewer 2: ```suggestion
				decline

```

# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Successors: First Contact Exploration: Kua-Oa-Aava"
	minor
	source "Kua-Oa-Aava"
	to offer
		has "Successors: First Contact 1: done"
		not "Successors: First Contact 2: done"
	on offer
		conversation
			`You watch for a couple of minutes as the spaceport crew here attempt to find a fuel adapter that fits your ship. After some time, a group of military officers is called over; they scan the side of the <ship> thoroughly and input their results into a datapad. Eventually the leader produces another one of the Successors' morphic metal spheres; they briefly connect it to the datapad and it rapidly squishes into a shape that lets it interface with both your ship and the fuel hose.`

> 				decline

Author: I suppose that First Last could observe this through a window or something, so I think that's a fair suggestion.

			`	"From where on the map dost thou hail, captain?" Nnesa roughly gestures to an area of their map that you recognize as human space.`
			choice
				`	(Indicate Rutilicus.)`
					goto rutilicus
				`	(Indicate Sol.)`
					goto sol
				`	(Gesture broadly to the expanse of space depicted on the map.)`
			`	"Oh!" says Nnesa, "Thou art a scion of the constellations, first and foremost to the stars? 'Tis fair. Thou hast a galaxy to wander, a nearly endless sky. The things that there must be to see---"`
				goto interrupt
			label rutilicus
			`	"Oh!" says Nnesa, "I am not sure I know the name for that one, captain. What dost thou call thy home?"`
			choice
				`	"Rutilicus."`

> 					goto rutilicus2
> 			label rutilicus2

Reviewer 4: ```suggestion
```

			`	Nnesa asks you to repeat the word a second time, slowly working their way through your pronunciation and transcribing a short sequence of flowing letters on their map. "'Tis a pleasant name," they say.`
				goto interrupt
			label sol

> 			`	"Oh!" says Nnesa, "Sol." They quietly sound their way through the word a couple times, trying to get it right. "Our name for it is Saaqa-Viirja. 'Twas one of the northernmost extents of our Predecessors' voyages. Ah. But thou know'st this already, for sure."`

Reviewer 4: The Hai live far to the north from Sol, even if we count the wormhole. How come they are familiar with them?

> 				goto interrupt

Reviewer 4: ```suggestion
```

			label interrupt
			`	The two of you are interrupted by Jaasora, who needs Nnesa's help to pack the now-deflating tent back in its cart, and once that is done they indicate that they are ready to return home.`
	on complete
		"successor jobs" += 2
		payment 78000
		conversation
			`	Nnesa and Jaasora have continued to marvel at the stars on the way back home, in addition to spending a substantial amount of time poring over their mutual collection of photos, drawings, and maps from Maspa-Mavra. "Our deepest thanks, captain," Jaasora says to you once you have landed in the spaceport as Nnesa wrestles with the tent and cart. "'Twas a true and joyful voyage." Nnesa calls them over; the tent clearly requires at least two to handle. Jaasora presents you with your payment of <payment> and goes to help, and the two of them maneuver their equipment out of your cargo bay before shouting a hasty but good-humored farewell.`

mission "Successors: Graffiti"
	minor
	source
		government "Successor"
	to offer
		has "known to the successors"
		random < 15
	on offer
		conversation

> 			`Shortly after you disembark the <ship> for the spaceport on <source>, you hear a quiet but noticeable clattering from behind you near where your ship is being refueled.`

Reviewer 4: Should this be a `landing` mission?

Author: I figured that First Last may not necessarily disembark their ship every time they land at a spaceport. This mission relies on FL disembarking (either to notice and interact with the "repair crew" or to give the "crew" sufficient time to graffiti the ship), so I think keeping it as a spaceport mission is more appropriate under that reasoning.

			`	Waves of color run up the bodies of the others, and the tips of their arms undulate gently.`
			choice
				`	"I've never seen a solution like that before. How does it work?"`
					goto how
				`	"I'm sorry to interrupt in that case. Carry on."`
					goto fooled

			label how
			`	The group look around at each other before the same one replies again. "The... ah... solution seeps into the cracks and strains within the hull and when it dries it seals them. Beyond that, we do not know; we just apply it as directed by the port authority."`
			`	One of the other ones pipes up, cheerfully adding, "We don't know anything!" before the others give them a dark look.`
			choice
				`	"And it's so colorful too! Are you sure it's not a paint of some sort?"`

> 					goto colorful

Reviewer 4: ```suggestion
```

				`	"Fascinating. Carry on."`
					goto fooled

> 			label colorful

Reviewer 4: ```suggestion
```

			`	"Oh, that's for... ah... safety. Yes, 'tis very dangerous to touch if thou'r'nt trained, so 'tis bright in color as a warning." None of the three are wearing any sort of protective equipment that you can see, and by now most of the group have changed color to what you might characterize as a mirthful shade of pink. This is something which one or two are clearly trying to suppress, their skin shifting in gradients in an effort to maintain their previous dull purple hue. One of the others is trying to hide a bag filled with more cans of "hull solution" behind their arms.`
			choice
				`	"Well, if you're working on behalf of the port, then surely I can ask you to remove it as well."`

> 					goto called

Reviewer 4: ```suggestion
```

				`	"Nice try, but you and I both know what's going on here."`

> 					goto called

Reviewer 4: ```suggestion
```

				`	"I think my ship's hull has been well enough repaired. It might be best for you to attend to whatever other vessels are on your list."`
					goto complicit
				`	"Thanks for answering my questions. Please carry on."`
					goto fooled

> 			label called

Reviewer 4: ```suggestion
```

> 			`	The trio seem rather deflated at hearing this, but after quite a bit of milling amongst themselves eventually relent and agree to remove the solution so long as you don't inform the port authority about their actions. While they put good effort into removing it---or at least act the part---they can't seem to scrub away the most dried-on parts, and you eventually let them go after it becomes apparent that some of the paint has permanently bonded to the hull.`

Reviewer 4: ```suggestion
			`	The trio seem rather deflated at hearing this, but after quite a bit of milling amongst themselves eventually relent and agree to remove the solution so long as you don't inform the port authority about their actions. While they put good effort into removing it - or at least act the part - they can't seem to scrub away the most dried-on parts, and you eventually let them go after it becomes apparent that some of the paint has permanently bonded to the hull.`
```

				goto questions
			label end
			action
				clear "Successors: successor"
				clear "Successors: predecessor"
				clear "Successors: cluster"
	on complete
		"successor jobs" += 2
		payment 200000
		conversation
			`	Kaatiora remains excited and talkative on the way back, though marginally less so now that they have exhausted the totality of your knowledge of the biological sciences and have the analysis of their recently-gathered data to occupy them.`
			`	Upon landing at <destination>, they disembark from your ship, once again carrying their excess of equipment, but not before giving you your payment of <payment> and once again offering their thanks for transporting them.`

> mission "Successors: Buskers"

Reviewer 2: That's basically what it is, just put it the side missions file.

				decline
			label unappreciative
			`	You return to your ship, gently extricating yourself from the throng of spellbound Successors, who continue to shimmer and sway in the distance until the performance eventually ends and they disperse to go their separate ways.`
				decline

mission "Successors: Stargazing 1"
	minor
	name "Transport Successor stargazers"
	description "Transport a pair of Successor stargazers to <planet stopovers> and return them home to <destination> by <date>."
	source
		attributes "successor"
		not attributes "crime" "tourism" "uninhabited"
	stopover "Khosa-Kella-Oa"
	to offer
		has "known to the successors"
		random < 40
	to fail
		has "Successors: Unified Defense: offered"
	passengers 2
	cargo "Vacuum Tents" 1
	deadline 20
	on offer
		conversation
			`While in the spaceport on <origin>, you notice a pair of tired-looking Successors lugging around a cart of a curious folded material, taking turns dragging their apparently heavy luggage over the floor. Both perk up substantially when they see you and your ship, however, and pull their equipment towards you with renewed vigor, finally getting close enough to speak.`

> 			`	The larger of the two begins, "Oh! Thour't that recent alien, no? 'Tis a gorgeous ship thou commands-"`

Author: I prefer the current wording to keep the rhythm of the sentence: the player is "recent" in that they have recently arrived in Successor space.

					goto ignored
			`	Upon approaching the source of the clattering, you catch a trio of Successors in the process of smearing the hull of the <ship> with globs of a colorful, foul-smelling solution. It is so pungent you can barely approach them without retching, but the Successors seem not to notice, or not to care.`
			choice
				`	"What are you doing to my ship?"`
					goto what
				`	"Excuse me, are you all part of the repair crew?"`
			`	The three Successors startle at the sound of your voice, arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "Yes, correct thou art! We've orders to apply this ... hull solution to repair a spot of damage here."`
			`	Waves of color run up the bodies of the others, and the tips of their arms undulate gently.`
			choice
				`	"I've never seen a solution like that before. How does it work?"`
					goto how
				`	"I'm sorry to interrupt in that case. Carry on."`
					goto fooled
			label what

> 			`	The three Successors startle at the sound of your voice, their arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "We... ah... we're part of the repair crew! We've orders to apply this... hull solution to repair a spot of damage here."`

Author: I assume that ships accumulate wear and tear over travel that might require occasional repair; even if this isn't the case, the Successor kids here aren't meant to be especially convincing as they are making up their lies on the spot.

				decline
			label fooled
			`	Upon returning from the spaceport some time later, you find that the "repair crew" has covered an area on the side of the <ship> in their colorful and noxious solution. The pattern forms some words in the Successor script placed alongside a rather crude depiction of your face. The translation device struggles to interpret the non-standard glyphs, but it eventually resolves the text as "'Erjn'-Fool! No quills, no brain!" The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`
				decline

mission "Successors: Unified Defense"
	invisible
	landing
	infiltrating

> 	to offer

Reviewer 4: This should also check for the successor governments.

		or
			"reputation: Successor" < -10
			"reputation: New Houses" < -10
			"reputation: Old Houses" < -10
	to complete
		never
	on offer
		"reputation: Successor" = -999
		"reputation: Old Houses" = -999
		"reputation: House Chydiyi" = -999
		"reputation: House Myurej" = -999
		"reputation: House Sioeora" = -999
		"reputation: New Houses" = -999
		"reputation: House Aqrabe" = -999
		"reputation: House Kaatrij" = -999
		"reputation: House Seineq" = -999
		event "successors: unified defense"
		conversation
			branch uninhabited
				has "flagship planet attribute: uninhabited"
			branch "populated language"
				has "language: Successor"

> 			`You notice after landing here that the spaceport crew is making no effort to refuel or repair your ship, and that a large contingent of Successor military officers is gathering around your landing pad, their weapons trained on your vessel. As several interceptors begin to hover over your ship as well with their weapons ready, you are forced to take off.`

Reviewer 4: Can this trigger without first contact?

				launch
			label "populated language"
			`You notice after landing here that the spaceport crew is making no effort to refuel or repair your ship, and that a large contingent of Successor military officers is gathering around your landing pad.`
			`	You soon receive a transmission routed through the local spaceport. "Thy recent actions against our people make it very clear that we were unwise to permit thee access to our space. The Houses have decreed in council that thou art an enemy, and thou shalt be treated accordingly hereforth."`
			choice
				`	"It was an accident!"`
					goto accident
				`	"I apologize for the transgression. I won't do it again."`
					goto "never again"
				`	"They deserved it."`
					goto "deserved it"
			label uninhabited
			branch "uninhabited language"
				has "language: Successor"

> 			`Shortly after landing here, you receive a long-range transmission from the Successors. Though you can't make sense of any of the words, you get the distinct impression that they are angry at you.`

Reviewer 4: This also needs guarding against the first contact.
